### DISCUSSION TITLE
Metrics: age of image / date pulled

### DISCUSSION THREAD

**Discussion by Nils98Ar** [aquasecurity/trivy-operator#1751] [Help & Support]
Hi,

is there any way to get metrics about the age of container images or the date when it was pulled?

Best regards
Nils

**Comment by chen-keinan** [comment]
@Nils98Ar any idea from where to grab the info ? is it supported for all registries (ACR, GCR, Artifactory, DockerHub and etc.)

**Reply by Nils98Ar** [reply]
I would be happy with the local creation/pull date. At least this information seems to available on the nodes via CRI API (tested with K8s 1.27):
```
crictl inspecti <local image> | jq -r '.info.imageSpec.created'

```
My goal is to identify pods for which the update automation is missing or not working.
For example pods that are always recreated with an old local image (regarding local creation/pull date)  because a bad tag choice or pods that haven't been recreated for a long time.

As the kubelets garbage collection deletes unused images I think that old images (regarding local creation/pull date) still being present on the node are a good indicator.
